src/pages/TabHome.tsx

```diff
@@ -199,12 +200,12 @@ const TabHome = (props: HomeProps) => {
                 isDateEnabled={(date: string) => {
```

Comment:
I suggest parsing here so that in ```getActiveDates``` there are only calculations.
And change  ```date: string``` argument to ```isoDateString```.

Reply:
Agree, fixed